.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+version: 2.1
+
+workflows:
+  pr-build-and-test:
+    jobs:
+      - test_notify_queue_consumer:
+          name: test_notify_queue_consumer
+      - build_notify_queue_consumer:
+          name: build_notify_queue_consumer
+          requires: [test_notify_queue_consumer]
+
+orbs:
+  notify_queue_consumer:
+    commands:
+      install_aws_cli:
+        steps:
+          - run:
+              name: Install AWS CLI
+              command: sudo pip3 install awscli --upgrade
+      ecr_login:
+        steps:
+          - run:
+              name: Login to ECR
+              command: |
+                temp_role=$(aws sts assume-role --role-arn arn:aws:iam::311462405659:role/sirius-ci --role-session-name ci)
+                export AWS_ACCESS_KEY_ID=$(echo $temp_role | jq .Credentials.AccessKeyId | xargs)
+                export AWS_SECRET_ACCESS_KEY=$(echo $temp_role | jq .Credentials.SecretAccessKey | xargs)
+                export AWS_SESSION_TOKEN=$(echo $temp_role | jq .Credentials.SessionToken | xargs)
+                eval $(aws ecr get-login --no-include-email --region=eu-west-1)
+    executors:
+      python:
+        docker: [image: circleci/python]
+
+jobs:
+  test_notify_queue_consumer:
+    steps:
+      - checkout
+      - run:
+          name: Build container
+          command: docker-compose build
+      - run:
+          name: Run Linting Checks
+          command: docker-compose run --rm lint
+       - run:
+          name: Run Static Analysis
+          command: docker-compose run --rm phpstan
+      - run:
+          name: Run Unit Tests
+          command: docker-compose run --rm test
+      - store_test_results:
+          path: build
+
+  build_notify_queue_consumer:
+    executor: notify_queue_consumer/python
+    parameters:
+      container_repo_url:
+        type: string
+        default: 311462405659.dkr.ecr.eu-west-1.amazonaws.com/notify_queue_consumer
```

Comment:
`notify_queue_consumer` should be `notify-queue-consumer`

Reply:
Done